### PR TITLE
Story/configure rubocop

### PR DESCRIPTION
**Pull request by jicardu**
## Summary

- Install Rubocop
- Fix offenses for /app
- Add .rubocop.yml to ignore warnings

## Trello Card

https://trello.com/c/keMDwruN/8-configurar-rubocop

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Documentation:
+   Enabled: false
```
Comment (marcegl91) [inline .rubocop.yml]: What does happen when you run the command that the card mentions for rubocop? ie: `bundle exec rubocop app spec -R`

Comment (jicardu) [reply .rubocop.yml]: When I run `invalid option: -R` it responds: "invalid option: -R".

Without -R parameter: 
> Inspecting 7 files
> .......
>7 files inspected, no offenses detected

Comment (marcegl91) [reply .rubocop.yml]: Exactly. The thing is the -R parameter is now deprecated, but we still needthe functionality that were provided by it. The -R modifier enabled the Rails cops of Rubocop. Cops are the rules that Rubocop has for checking issues in your code:

https://github.com/rubocop-hq/rubocop/blob/master/manual/cops.md

So Rails cops were cops that checked for things that were Rails-specific. So now that we don't have the -R parameter, this checks are missing, so it's up to you to configure rubocop to have again this functionality.

File: .rubocop.yml [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+LineLength:
+   Max: 99
```
Comment (sebasdeldi) [inline .rubocop.yml]: EOF, not a big deal but still nice to fix. Check if your text editor can do this by default :)

File: app/channels/application_cable/channel.rb [hunk 3]
```diff
@@ -1,3 +1,5 @@
+# frozen_string_literal: true
```
Comment (sebasdeldi) [inline app/channels/application_cable/channel.rb]: Just for information this is default in Ruby 3.0

File: .rubocop.yml (hunk 1 above)
Comment (jicardu) [reply .rubocop.yml]: maybe using this gem: `'rubocop-rails'` ?

**Pull request merged**